tools/test/examples/update.py

```diff
@@ -72,113 +94,167 @@ def upgrade_single_example(example, tag, directory):
     example - json example object containing the GitHub repo to update.
     tag - GitHub tag corresponding to a version of mbed-os to upgrade to.
     directory - directory path for the example.
+    ref - SHA corresponding to the supplied tag
     returns - True if the upgrade was successful, False otherwise.
     
     """
-    print("Upgrading single example at path '%s'" % directory)
     cwd = os.getcwd()
     os.chdir(directory)
     
-    return_code = None
+    return_code = False
     
-    # Change directories to the mbed-os library
-    if not os.path.exists('mbed-os'):
-        print("'mbed-os' directory not found in the root of '%s'" % directory)
-        print("Ignoring and moving on to the next example")
-        os.chdir(cwd)
+    if os.path.isfile("mbed-os.lib"):
+        os.system("mv mbed-os.lib mbed-os.lib_bak")
```

Comment:
`mv` is not a valid command on Windows, can you use [os.rename](https://docs.python.org/2/library/os.html#os.rename) instead please?